docs/core_plugin_conversion.md

```diff
@@ -0,0 +1,88 @@
+# Core annotation to attribute conversion
+
+To convert a plugin in core from an annotation to an attribute, you need to do the following:
+
+## Install drupal-rector
+```bash
+composer require --dev palantirnet/drupal-rector
+```
+
+## Configure drupal-rector (rector.php)
+
+Create the folling file in the root of core. In this example we are converting the `ContentEntityType` and `ConfigEntityType` annotations to attributes. If you want to convert other annotations, you will need to configure the `AnnotationToAttributeRector` with the appropriate `AnnotationToAttributeConfiguration` objects.
+
+```php
+<?php
+declare(strict_types=1);
+
+use Rector\Config\RectorConfig;
+
+return static function(RectorConfig $rectorConfig): void {
+  $rectorConfig->ruleWithConfiguration(\DrupalRector\Drupal10\Rector\Deprecation\AnnotationToAttributeRector::class, [
+
+    // Setting both introduce and remove version to 10.x means the comments are not kept. Which is good for core. ;) 
+    new \DrupalRector\Drupal10\Rector\ValueObject\AnnotationToAttributeConfiguration('10.0.0', '10.0.0', 'ContentEntityType', 'Drupal\Core\Entity\Attribute\ContentEntityType'),
+    new \DrupalRector\Drupal10\Rector\ValueObject\AnnotationToAttributeConfiguration('10.0.0', '10.0.0', 'ConfigEntityType', 'Drupal\Core\Entity\Attribute\ConfigEntityType'),
+  ]);
+
+  $rectorConfig->autoloadPaths([
+    './lib',
+    './modules',
+    './profiles',
+    './themes'
+  ]);
```

Comment:
These settings and you're in `core` but the script execution looks like it's from the directory above

Reply:
You're right